### PR TITLE
Add dependency.

### PR DESCRIPTION
**Pull request by asmab89**

### CODE REVIEW

File: e2e-tests/spring-graphql/pom.xml [hunk 1]
```diff
@@ -49,6 +49,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+            <version>2.5.0</version>
```
Comment (arcuri82) [inline e2e-tests/spring-graphql/pom.xml:55]: recall [https://github.com/EMResearch/EvoMaster/blob/master/docs/for_developers.md#maven-dependency-version](). version declaration should be in root pom. if u are not sure how Maven work, we can chat about this next meeting

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: e2e-tests/pom.xml [hunk 2]
```diff
@@ -32,6 +32,11 @@
                 <scope>test</scope>
                 <type>test-jar</type>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-validation</artifactId>
+                <version>2.5.0</version>
+            </dependency>
```
Comment (arcuri82) [inline e2e-tests/pom.xml:39]: this is not the main root pom file. furthermore, the version should be `${springboot.version}` as all other dependencies

Comment (asmab89) [reply e2e-tests/pom.xml]: Done

File: e2e-tests/spring-graphql/pom.xml (hunk 1 above)
Comment (asmab89) [reply e2e-tests/spring-graphql/pom.xml]: Done

**Pull request merged**